source/includes/_people.md

```diff
@@ -14,12 +13,169 @@ curl -X POST -H "Content-Type: application/json" -H "Cache-Control: no-cache" -d
     "page" : 1,
     "person_titles" : ["sales manager", "engineer manager"]
 }' "https://api.apollo.io/v1/mixed_people/search"
-```    
-    
+``` 
+
+> Sample response:
+
+```json 
+{
+    "breadcrumbs": [
+        {
+            "label": "Titles",
+            "signal_field_name": "person_titles",
+            "value": "sales manager",
+            "display_name": "sales manager"
+        },
+        {
+            "label": "Titles",
+            "signal_field_name": "person_titles",
+            "value": "engineer manager",
+            "display_name": "engineer manager"
+        },
+        {
+            "label": "Company Domains",
+            "signal_field_name": "q_organization_domains",
+            "value": [
+                "apollo.io",
+                "google.com"
+            ],
+            "display_name": "apollo.io, google.com"
+        }
+    ],
+    "partial_results_only": false,
+    "partial_results_limit": 10000,
+    "pagination": {
+        "page": 1,
+        "per_page": 10,
+        "total_entries": 1339,
+        "total_pages": 134
+    },
+    "contacts": [],
+    "people": [
+        {
+            "id": "57e1876fa6da987d8411c507",
+            "first_name": "Abhilash",
+            "last_name": "Mohanty",
+            "name": "Abhilash Mohanty",
```

Comment:
Do not give an example that is NOT an Apollo employee. Use either Tim or my profile